usecase/transaction_usecase.py

```diff
@@ -30,14 +35,18 @@ def get_transaction_details(self, get_transaction_details_in: GetTransactionDeta
             is_fee_greater_than_default = ticket_price > min_ticket_price_for_default
             transaction_fee = (P * transaction_fee_percentage) if is_fee_greater_than_default else default_fee
 
+        else:
+            return JSONResponse(status_code=422, content={'message': 'Invalid payment method was passed.'})
+
         equation = Eq(P - transaction_fee - (transaction_fee * vat), ticket_price)
 
         # Solve the equation for P
-        total_price = solve(equation, P)[0]
-        transaction_fee = total_price - ticket_price
+        total_price = solve(equation, P)[0] + platform_fee
+        transaction_fee = total_price - ticket_price - platform_fee
```

Comment:
The `platform_fee` should just be added as an addition to the `ticket_price`. This is done so that the transaction fee also accounts for the `platform_fee`.